ajax/libs/material-kit/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "material-kit",
+  "version": "1.1.0",
+  "description": "A Badass Bootstrap UI Kit based on Material Design",
+  "filename": "js/material-kit.min.js",
+  "homepage": "https://www.creative-tim.com/product/material-kit",
+  "author": "creative-tim",
+  "keywords": [
+    "material",
+    "bootstrap",
+    "design",
+    "google",
+    "twitter"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https+git://github.com/creativetimofficial/material-kit.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/creativetimofficial/material-kit.git",
+    "fileMap": [
+      {
+        "basePath": "assets",
+        "files": [
+          "+(js|css)/material-kit*"
```

Comment:
We also need img/arrow-left.png and img/arrow-right.png. Refer https://github.com/creativetimofficial/material-kit/blob/master/assets/css/material-kit.css#L2775 and https://github.com/creativetimofficial/material-kit/blob/master/assets/css/material-kit.css#L2778.